dev-python/gitdb2/gitdb2-2.0.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GitDB is a pure-Python git object database"
+HOMEPAGE="
+	https://github.com/gitpython-developers/gitdb
+	https://pypi.python.org/pypi/gitdb2"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	!dev-python/gitdb[${PYTHON_USEDEP}]
+	>=dev-python/smmap2-2.0.1[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+# Testsuite require files from a git repo
```

Comment:
Well actually I don't understand what exactly fails. Executing `git init` is a good starting point. However, copying the packs `cp gitdb/test/fixtures/* .git/` does not resolves all errors:

```
F.....Endurance run: verify streaming of objects (crc and sha)
PDB: verified 0 objects (crc=0) in 0.000073 s ( 0.000000 objects/s )
PDB: verified 0 objects (crc=1) in 0.000001 s ( 0.000000 objects/s )
..PDB: looked up 0 shas by index in 0.000060 s ( 0.000000 shas/s )
PDB: looked up 0 sha in 0 packs in 0.000000 s ( 0.000000 shas/s )
PDB: Obtained 10000 object INFO by sha in 0.000001 s ( 13981013333.333334 items/s )
PDB: Obtained 10000 object STREAM by sha in 0.000000 s ( 20971520000.000000 items/s )
PDB: Obtained 5000 streams by sha and read all bytes totallying 0 KiB ( 0.000000 KiB / s ) in 0.000000 s ( 10485760000.000000 streams/s )
.PDB Streaming: Got 1000 streams by sha in in 0.000049 s ( 20560313.725490 streams/s )
E.Creating  data ...
Done (in 0.989707 s)
Added 50000 KiB (filesize = 17294 KiB) of  data to loose odb in 0.750052 s ( 66662.046965 Write KiB / s)
Read 50000 KiB of  data at once from loose odb in 0.175808 s ( 284401.241131 Read KiB / s)
Read 50000 KiB of  data in 512 KiB chunks from loose odb in 0.158875 s ( 314713.791779 Read KiB / s)
Creating random  data ...
Done (in 13.076944 s)
Added 50000 KiB (filesize = 43182 KiB) of random  data to loose odb in 1.651726 s ( 30271.373223 Write KiB / s)
Read 50000 KiB of random  data at once from loose odb in 0.285888 s ( 174893.399483 Read KiB / s)
Read 50000 KiB of random  data in 512 KiB chunks from loose odb in 0.251156 s ( 199079.194932 Read KiB / s)
....S........
======================================================================
ERROR: test_pack_writing (gitdb.test.performance.test_pack_streaming.TestPackStreamingPerformance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/gitdb-2.0.2/gitdb/test/lib.py", line 72, in wrapper
    return func(self, *args, **kwargs)
  File "/tmp/gitdb-2.0.2/gitdb/test/performance/test_pack_streaming.py", line 58, in test_pack_writing
    PackEntity.write_pack((pdb.stream(sha) for sha in pdb.sha_iter()), ostream.write, object_count=ni)
  File "/tmp/gitdb-2.0.2/gitdb/pack.py", line 992, in write_pack
    "Expected to write %i objects into pack, but received only %i from iterators" % (object_count, actual_count))
nose.proxy.ValueError: Expected to write 1000 objects into pack, but received only 0 from iterators
-------------------- >> begin captured logging << --------------------
root: INFO: You can set the GITDB_TEST_GIT_REPO_BASE environment variable to a .git repository of your choice - defaulting to the gitdb repository
--------------------- >> end captured logging << ---------------------

======================================================================
FAIL: test_reading (gitdb.test.db.test_git.TestGitDB)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/gitdb-2.0.2/gitdb/test/db/test_git.py", line 29, in test_reading
    assert gdb.size() >= ni
nose.proxy.AssertionError: 
-------------------- >> begin captured logging << --------------------
root: INFO: You can set the GITDB_TEST_GIT_REPO_BASE environment variable to a .git repository of your choice - defaulting to the gitdb repository
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 24 tests in 18.335s

FAILED (SKIP=1, errors=1, failures=1)
```